common.tf

```diff
@@ -33,6 +33,8 @@ locals {
   enable_apiary_s3_log_management = var.apiary_log_bucket == "" ? true : false
   apiary_s3_logs_bucket           = local.enable_apiary_s3_log_management ? "${local.apiary_bucket_prefix}-s3-logs" : ""
   apiary_s3_hive_logs_bucket      = local.enable_apiary_s3_log_management ? "${local.apiary_s3_logs_bucket}-hive" : ""
+  apiary_system_bucket            = "${local.apiary_bucket_prefix}-system"
```

Comment:
This relies on us deciding on the name in the [other PR](https://github.com/ExpediaGroup/apiary-metastore-docker/pull/77) first...